qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -134,19 +132,64 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
             }
             hasNetworkViolation = results.any { isNetworkViolation(it) }
             hasProcessViolation = results.any { isProcessViolation(it) }
+            // TODO: Remove debug log
+            log.info "Found violations in Splunk: \n${results}"
+            log.info "hasNetworkViolation: ${hasNetworkViolation}\nhasProcessViolation: ${hasProcessViolation}"
+            // TODO: /Remove debug log
             if (hasNetworkViolation && hasProcessViolation) {
                 log.info "Success!"
                 break
             }
         }
 
+        // After we know that violations arrived, manually kick off search that converts them into alerts
```

Comment:
Let's turn that into info log with a milestone message - violations arriving looks like an important milestone to reach